src/server.js

```diff
@@ -100,11 +101,14 @@ io.on('connection', socket => {
             console.log(`Apple ${appleCollision.appleId} was eaten!`);
             eatenApples.push(appleCollision.appleId);
             apples.splice(apples.findIndex(apple => apple.id === appleCollision.appleId), 1);
+            playerState.score += appleScoreValue;
+            socket.emit('update score', playerState.score);
```

Comment:
We will need to broadcast that event if we want other players to see the score, but we can do that later if/when we do a score list.

Reply:
I think the idea is that's handled by the "update players" event below.